haas/ext/switches/dell.py

```diff
@@ -113,6 +114,9 @@ def disable_native(self, vlan_id):
         self._sendline('sw trunk native vlan none')
 
     def disconnect(self):
+        if cfg.has_option('dell', 'save'):
+            if cfg.getboolean('dell', 'save'):
```

Comment:
I was just worried if we don't have that section then cfg.getboolean returns an error. But now that I tested it it doesn't break the flow of the program. So I'll change it to use `and`.